pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -137,14 +138,28 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *v1.Pod, attemp
 	return podSandboxConfig, nil
 }
 
+// If PodSandboxSeccomp is enabled, overwrites pod level profile with SeccompProfileRuntimeDefault
+// This allows for pod level profiles to be applied only for user-containers, making them less privileged.
+func (m *kubeGenericRuntimeManager) getSandboxSeccompProfilePath(pod *v1.Pod) string {
+	// TODO: Allow for json seccomp profiles to be sent across to CRI, alongside with the "no-new-privileges" flag.
+	// This would allow for defaultPodSandboxSeccomp to be used instead.
+	profileName := m.getSeccompProfileFromAnnotations(pod.Annotations, "")
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodSandboxSeccomp) {
+		profileName = core.SeccompProfileRuntimeDefault
```

Comment:
@tallclair this deals with non dockershim (i.e. cri). The API will have to be changed in order to support a full seccomp profile to be sent across for sandboxes, together with the no-new-privileges flag. This can be a target for when this becomes Beta/GA.

Alternatively, the least privilege seccomp profile can be created on disk, and then used for both dockershim and CRI using the existing `localhost/sandbox-profile` syntax.

Reply:
I reverted this change for non-dockershim. I think that may be better handled as a separate PR.